### PR TITLE
Improve search results

### PR DESCRIPTION
**Pull request by geroyche**
As a heavy discogs user with a collection of more than 2400 items, i love this. However, search matched only a fraction of my collection.
One issue is an apparent bug in Spotify's Search API, or at least an undocumented behaviour. Artist names or release titles with single quotes yield no results. Simply removing them works. You can verify this in Spotify's Developer Console with Artist "Vex'd", or the Album "If you're into it, I'm out of it" by "Christoph de Babalon".

I also have 33 releases that end with "E.P." rather an "EP", and 8 that end on "E.P". You'll also find many ending with "L.P." on Discogs.

Please note that I didn't test the resulting code, but i verified the improved search results in Spotify's Developer Console.

### CODE REVIEW

File: js/export.js [hunk 1]
```diff
@@ -459,7 +459,7 @@ function exportMultipleMatches() {
                 imageURL = album.images[0].url;
             }
 
-            $('#spotifyDiv').append('<div><img src="' + imageURL + '" width="20%" style="display:inline-block; margin:10px; vertical-align:top"><div style="display:inline-block; width:70%"><h4>' + album.name + '</h4><button id="' + albumID + ' ' + imageURL + '" type="button" class="btn btn-success" onClick = "saveAlbumFromMulti(this.id)"><span class="icon-checkmark"></span> Choose this</button></div></div>');
+            $('#spotifyDiv').append('<div><img src="' + imageURL + '" width="20%" style="display:inline-block; margin:10px; vertical-align:top"><div style="display:inline-block; width:70%"><h4>' + name + '</h4><button id="' + albumID + ' ' + imageURL + '" type="button" class="btn btn-success" onClick = "saveAlbumFromMulti(this.id)"><span class="icon-checkmark"></span> Choose this</button></div></div>');
```
Comment (geroyche) [inline js/export.js:462]: var name from line 454 was unused. figured i might as well use it...

Comment (lim93) [reply js/export.js:462]: Oops ;-)

**Comment by lim93** [comment]
Thank you very much for your contribution! Your changes are looking good to me, I will test them out and release a new version shortly.

**Comment by lim93** [comment]
I just released a new version, including your changes and some minor improvements. 
Turns out that `replace('\'', '')` only replaces the first instance of a single quote in a String, so it wouldn't work for titles like "_If you're into it, I'm out of it_". I changed that to `replace(/'/g, '')`, which replaces all instances. 
Also, the search API was still struggling with "_'n'_" in a title, for example "_Twists'n'Turns_". I tried replacing it with " _n_ " as a workaround, which seems to work. 

Thanks again for your contribution!

**Pull request merged**